expansion/src/test/java/no/unit/nva/expansion/ResourceExpansionServiceTest.java

```diff
@@ -332,6 +323,41 @@ void shouldCopyAllPubliclyVisibleFieldsFromMessageToExpandedMessage(Class<? exte
         assertThat(regeneratedMessage, is(equalTo(message)));
     }
 
+    @DisplayName("should not update ExpandedTicket status when ticket status is not pending")
+    @ParameterizedTest
+    @MethodSource("no.unit.nva.publication.ticket.test.TicketTestUtils#ticketTypeAndPublicationStatusProvider")
+    void shouldNotUpdateExpandedTicketStatusWhenTicketStatusIsNotPending(Class<? extends TicketEntry> ticketType,
```

Comment:
Not sure i understand the purpose of this test. We want to update expanded ticket status with the other statuses as well, don't we?